Missing NeutralResourcesLanguageAttribute

After adding Automapper 6.1.1. to an UWP I get compilation warning: 
`warning MSB3817: The assembly "............\.nuget\packages\AutoMapper\6.1.1\lib\netstandard1.3\AutoMapper.dll" does not have a NeutralResourcesLanguageAttribute on it. To be used in an app package, portable libraries must define a NeutralResourcesLanguageAttribute on their main assembly (ie, the one containing code, not a satellite assembly).
`
### Source/destination types
Compilation warning : Not applicable

### Mapping configuration
Compilation warning : Not applicable

### Version: x.y.z
6.1.1

### Expected behavior
No warning to be produced

### Actual behavior
Warning MSB3871 produced
.....\.nuget\packages\AutoMapper\6.1.1\lib\netstandard1.3\AutoMapper.dll : warning MSB3817: The assembly "............\.nuget\packages\AutoMapper\6.1.1\lib\netstandard1.3\AutoMapper.dll" does not have a NeutralResourcesLanguageAttribute on it. To be used in an app package, portable libraries must define a NeutralResourcesLanguageAttribute on their main assembly (ie, the one containing code, not a satellite assembly).

### Steps to reproduce
1. Create new Blank UWP solution
2. Add Automapper 6.1.1.
3. Build

or you can just use example solution at : [https://github.com/milanio/repro-MSB3817-automapper](https://github.com/milanio/repro-MSB3817-automapper)

--- a/src/AutoMapper/AssemblyInfo.cs
+++ b/src/AutoMapper/AssemblyInfo.cs
@@ -1,4 +1,6 @@
 ﻿using System;
+using System.Resources;
+
 #if NET45 || NET40
 using System.Security;
 #endif
@@ -7,3 +9,5 @@ using System.Security;
 #if NET45 || NET40
 [assembly: AllowPartiallyTrustedCallers]
 #endif
+
+[assembly: NeutralResourcesLanguage("en")]

